src/components/Button/Button.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import styles from './Button.module.scss';
+import { Loader, LoaderSize } from '../Loader/Loader';
```

Comment:
лучше так, а если не получается, нужно сделать, чтобы получалось😋
```suggestion
import { Loader, LoaderSize } from '../Loader';
```

в данном случае нужно реэкспортить в components/Loader/index.ts ещё и LoaderSize 

некрит